src/modules/get-weather-async.js

```diff
@@ -0,0 +1,13 @@
+import renderWeather from './render-today';
```

Comment:
I'm again confused about naming, it is not clear what we render here: weather or today?
usually the import name should match somehow the name of module, for ex., `import renderToday from './render-today';` 
But here is an entering point and we should render not today but the wigget itself. So I would suggest to make it like `import initWeatherApp from './init-weather-app;` or something similar